requirements/dev.txt

```diff
@@ -256,7 +256,7 @@ pymongo==3.12.1
     #   -r requirements/base.txt
     #   -r requirements/test.txt
     #   edx-opaque-keys
-pyparsing==3.0.3
+pyparsing==2.4.7
```

Comment:
version is getting downgraded due to an internal constraint requirement of `packaging` package. This constraint has been removed and will be updated in the next release of `packaging` package.